.test.hawk0.env

```diff
@@ -0,0 +1,5 @@
+SMPC__DATABASE__URL=postgres://postgres:postgres@dev_db:5432/SMPC_dev_0
+SMPC__REQUESTS_QUEUE_URL=http://sqs.us-east-1.localhost.localstack.cloud:4566/000000000000/smpcv2-0-dev.fifo
+SMPC__NODE_HOSTNAMES='["0.0.0.0","hawk_participant_1","hawk_participant_2"]'
```

Comment:
need to pass local interface for grpc server for its own address due to the hawk main loading mechanics

Reply:
probably would be good 👍 would be best to just pass the real hostnames to al of them instead of having to override their own

Reply:
Done here: https://github.com/worldcoin/iris-mpc/pull/1038